dbc_changelog.md

```diff
@@ -0,0 +1,479 @@
+# Autonomoustuff User CAN DBC File
+
+The as_pacmod.dbc file is formatted as a Vector DBC file.  This file specifies changes to this file.
+
+## Change 005 Approved 2/22/2019
+
+### Requirements
+
+1. ID conflicts with messages in the DBC.
+
+### Changes
+
+1. Modify – Message: NOTIFICATION_CMD 
+   1. Message Properties: 
+      1. Address: 0x421.
+
+2. Modify – Message: LINEAR_ACCEL_RPT
+   1. Message Properties: 
+      1. Address: 0x419.
+
+3. Modify – Message: ANG_VEL_RPT
+   1. Message Properties: 
+      1. Address: 0x420.
+
+## Change 004 Approved 2/18/2019
+
+### Requirements
+
+1. Update motor reports to align with globe motor message formats.
+
+### Changes
+
+1.       Modify – Message: BRAKE_MOTOR_RPT_1 
```

Comment:
The additional spaces here are causing problems with the formatting. The MD interpreter things this is a "pre-formmated section" instead of a list. Please remove these from this point down in the document.